issues.go

```diff
@@ -81,7 +128,22 @@ func NewIssue(ctx context.Context, client *github.Client, settings *Settings) (i
 	}
 	io.WriteString(tempFile, "\n# Uncomment to assign labels\n")
 	for _, l := range labels {
-		fmt.Fprintf(tempFile, "# Label: %s\n", l)
+		// if labels are passed as command line input, uncomment them
+		labelMatch := false
+		if labelSlice != nil {
+			for _, labelInput := range labelSlice {
+				if labelInput == l {
+					fmt.Fprintf(tempFile, "Label: %s\n", l)
+					labelMatch = true
+					break
+				}
+			}
+			if labelMatch == false {
+				fmt.Fprintf(tempFile, "# Label: %s\n", l)
+			}
```

Comment:
What are your thoughs on making `labelSlice` a `labelSet`? that could help out here by doing:
```golang
if labelSet[l] {
    fmt.Fprintf(tempFile, "Label: %s\n", l)
    continue
}
fmt.Fprintf(tempFile, "# Label: %s\n", l)
```